rust/main/agents/relayer/src/relayer.rs

```diff
@@ -166,6 +169,15 @@ impl BaseAgent for Relayer {
             &settings,
             core_metrics.clone(),
             &chain_metrics,
+            db.clone(),
+        )
+        .await;
+
+        let dispatchers = Self::build_payload_dispatchers(
+            &settings,
+            core_metrics.clone(),
+            &chain_metrics,
+            db.clone(),
         )
         .await;
```

Comment:
Let's have a debug log message about how much time we spend initialising `dispatchers` as we do for other entities (mailboxes, validator announces, etc)